enthought_example/example_plugin.py

```diff
@@ -1,14 +1,16 @@
-from force_bdss.api import BaseExtensionPlugin, plugin_id
+from force_bdss.api import plugin_id
+from force_wfmanager.ui import UIExtensionPlugin
 
 from .example_notification_listener import ExampleNotificationListenerFactory
 from .example_mco import ExampleMCOFactory
 from .example_data_source import ExampleDataSourceFactory
 from .example_ui_hooks import ExampleUIHooksFactory
+from .example_contributed_ui import ExampleContributedUI
 
 PLUGIN_VERSION = 0
 
 
-class ExamplePlugin(BaseExtensionPlugin):
+class ExamplePlugin(UIExtensionPlugin):
```

Comment:
Making this ExamplePlugin a subclass of a `wfmanager` class kind of defies the purpose of several design choices made for the custom data views.

We should 
1. either revert this choice and do without the intermediate `UIExtensionPlugin` class,
2. or have a third example plugin (`ExampleUIPlugin`) which subclasses from `UIExtensionPlugin` to showcase this functionality.

I personally prefer (1), but I'm not sure I'll have time to be further involved in this

Reply:
For now, option 1) has been implemented so that the `ExamplePlugin` continues to showcase both `CustomUI` and `DataView` functionalities, but only requires access to the `force_wfmanager` upon calling either `get_contributed_uis` or `get_data_views` methods.